cabby/geo/walk_test.py

```diff
@@ -16,37 +16,46 @@
 import osmnx as ox
 import unittest
 from shapely.geometry.point import Point
+
 from cabby.geo import walk
+from cabby.geo import util
+from cabby.geo.map_processing import map_structure
 
 
 class WalkTest(unittest.TestCase):
 
     def setUp(self):
-        
+
         # Load map from disk.
-        self.map = map_structure.Map("Manhattan", 18, "/mnt/hackney/data/cabby/poi/")
-
-    def testSingleOutput(self):
-        start_point = Point(-73.982473, 40.748432)
-        end_point = Point(-73.98403, 40.74907)
-        list_points = walk.compute_route(start_point, end_point, self.map.poi_graph, self.map.nodes)
-
-        # Check that two points have arrived.
-        self.assertEqual(len(list_points), 2)
-
-        # Check that the correct points have arrived.
-        first_point = walk.tuple_from_point(list_points[0])
-        second_point = walk.tuple_from_point(list_points[1])
-        self.assertEqual(first_point, (40.749102, -73.984076))
-        self.assertEqual(second_point, (40.748432, -73.982473))
-
-        # Check that all points are in a bounding box.
-        eps = 0.0005
-        for point in list_points:
-            self.assertLessEqual(point.x, start_point.x + eps)
-            self.assertGreaterEqual(point.x, end_point.x - eps)
-            self.assertLessEqual(point.y, end_point.y + eps)
-            self.assertGreaterEqual(point.y, start_point.y - eps)
+        self.map = map_structure.Map(
+            "Bologna", 18)
+
+    def testRouteCalculation(self):
+        start_point = Point(11.3414, 44.4951)
+        end_point = Point(11.3444, 44.4946)
+        route = walk.compute_route(start_point, end_point, self.map.
+        nx_graph, self.map.nodes)
+
+        # Check the size of the route. 
+        self.assertEqual(len(route), 9)
+
+        # Check that the correct points are in the route.
+        first_point = util.tuple_from_point(route.iloc[0]['geometry'])
+        second_point = util.tuple_from_point(route.iloc[1]['geometry'])
+        self.assertEqual(first_point, (44.4946187, 11.344085))
+        self.assertEqual(second_point, (44.4947274, 11.343436))
```

Comment:
My previous question about using assertAlmostEqual here seems to have disappeared. Do you think it is okay to use assertEqual here?

Reply:
Okay to leave as is for now.